jst-nuke-init

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env node
+
+var exec = require('child_process').execSync
+var inquirer = require('inquirer')
+var clone = require('gitclone')
+var rimraf = require('rimraf')
+var replaceInFile = require('replace-in-file')
+var cd = process.chdir
+var GitHubApi = require('github')
+
+var github = new GitHubApi()
+
+var replace = function (replace, withWhat) {
+  replaceInFile.sync({
+    files: ['package.json', 'README.md', 'index.js'],
+    replace: replace,
+    with: withWhat
+  })
+}
+
+var errHandler = function (err) {
+  if (err) { console.error(err.stack) }
+}
+
+var print = function (data) {
+  console.log(data.stdout)
+  console.error(data.stderr)
+}
+
+var run = function (command) {
+  console.log(exec(command).toString())
+}
+
+var getInfo = function () {
+  return inquirer.prompt([
+    {
+      name: 'humanName',
+      message: 'Name of JSTransformer (capitalized properly)'
+    },
+    {
+      name: 'url',
+      message: 'URL to find more about the transformer'
+    },
+    {
+      name: 'username',
+      message: 'GitHub Username (for creating repo)'
+    },
+    {
+      name: 'password',
+      type: 'password',
+      message: 'GitHub Password (for creating repo)'
+    }
+  ]).then(function (answers) {
+    github.authenticate({
+      type: "basic",
+      username: answers.username,
+      password: answers.password
+    }, function () {console.log('authenticated with GitHub')})
+    answers.lowerName = answers.humanName.toLowerCase().replace(' ', '')
+    answers.transformer = `jstransformer-${answers.lowerName}`
+    return answers
+  })
+}
+
+var setup = function (answers) {
+  clone('jstransformers/boilerplate', {dest: answers.transformer}, function () {
+    cd(answers.transformer)
+    run('pwd')
+    rimraf.sync('.git')
+    github.repos.createForOrg({
```

Comment:
http://mikedeboer.github.io/node-github/#api-repos-createForOrg

Seems like a neat idea. Hadn't thought of doing all of this in the command line. Could creating it in the Org be an option we pass? May not want to create a repo for it all the time.